Formula/libtpms.rb

```diff
@@ -18,7 +19,7 @@ class Libtpms < Formula
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
-  depends_on "openssl@3"
+  depends_on "openssl@1.1"
```

Comment:
That doesn't seem like a good solution, I'd rather not downgrade dependencies.

Reply:
The other way may be to upgrade openssl for other dependencies. It might be risky since python3.9 uses openssl1.1, and it seems it does not support openssl3 yet.